charts/aws-efs-csi-driver/values.yaml

```diff
@@ -45,6 +45,10 @@ controller:
   tags: {}
     # environment: prod
     # region: us-east-1
+  # Add additional env to containers
+  env:
+    efsPlugin: []
```

Comment:
for consistency with EBS, this should just be `controller.env: []`

similarly, `controller.env.csiProvisioner: []` should be moved to `sidecars.csiProvisioner.env`

Reply:
ref:

https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/values.yaml#L65

https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/values.yaml#L13